app/views/teams/show.html.erb

```diff
@@ -43,6 +43,37 @@
   </tbody>
 </table>
 
+<% if can? :assign_membership_by_email, @team %>
+  <%= link_to t('teams.show.invite_user_to_team'),
+              '#',
+              data: {'toggle': 'modal', 'target': '#invite_user_by_email_modal'},
+              class: 'btn btn-primary' %>
+
+  <div class="modal fade" id="invite_user_by_email_modal" tabindex="-1" role="dialog" aria-labelledby="<%= t('teams.invite_user_to_team.title') %>" aria-hidden="true">
+    <div class="modal-dialog modal-sm" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="<%= t('helpers.links.close') %>"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title"><%= t('teams.invite_user_to_team.title') %></h4>
+        </div>
+        <form action="<%= assign_membership_by_email_team_path(@team) %>" method="post">
+          <%= hidden_field_tag :authenticity_token, form_authenticity_token %>
+          <div class="modal-body">
+            <div class="form-group">
+              <label for="invited-user-email" class="form-control-label"><%= t('activerecord.attributes.user.email') %></label>
+              <input type="email" class="form-control" id="invited-user-email" name="email">
+            </div>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-secondary" data-dismiss="modal"><%= t('helpers.links.cancel') %></button>
```

Comment:
Just for consistency reasons: Consider using `btn btn-default`, like we do for other buttons on show page.

Reply:
I totally agree with you in matters of consistency but we will lose some semantic information when using the `btn-default` class. Will pass that decision to @bakoe :-)

Reply:
What do you mean by "semantic information"?

Reply:
...that it's a "secondary button", no "default button" in a semantic sense.

Reply:
I just noticed that the version of Bootstrap currently used in the sport-portal – version 3 – does not even have the `btn-secondary` class. Sorry for that! 😂

I'll replace the class with `btn-default` (for the cancel button) and leave the `btn-primary` class for the submit button.